sdk/servicebus/pom.service.xml

```diff
@@ -10,5 +10,6 @@
   <version>1.0.0</version><!-- Need not change for every release-->
   <modules>
     <module>microsoft-azure-servicebus</module>
```

Comment:
This should be updated to have the profiles.
https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/eventhubs/pom.service.xml#L10-L45